docs/UserGuide.md

```diff
@@ -217,12 +217,16 @@ Examples:
 * `L add m/GET1020 n/lecture d/Mon 8:00-10:00`
 * `L add m/CS3233 n/tutorial d/MON 17:30-20:00`
 
-:warning: The `TYPE` must be one of the following: `lecture`, `tutorial`,
-`lab`, `recitation`, or `sectional`.
+:warning: The `TYPE` must be one of the following or its abbreviation:
+`lecture`/`lec`, `tutorial`/`tut`,
+`laboratory`/`lab`, `recitation`/`rec`, or `sectional`/`sec`.
```

Comment:
I think we should put the shorter type before the longer type!

Reply:
Mm, I think the original name should be mentioned before its abbreviation, but maybe it's just me.